app/domain/output_document.rb

```diff
@@ -58,6 +63,8 @@ def variant
   end
 
   def greeting
```

Comment:
`greeting` doesn't seem appropriate for this text. Perhaps `lead` would be better? (or `lede`)